README.md

```diff
@@ -1,4 +1,4 @@
-# CNCF TAG Contributor Strategy
+# CNCF TAG (Technical Advisory Groups) Contributor Strategy
```

Comment:
I have been taught that when introducing a new acronym, to use the full term first, and then provide the acronym as you will be using it throughout the rest of the document. Since that is too verbose for a heading, we can follow this advice for the first reference of the term in the next paragraph instead.

I recommend the following to improve readability:

```
# CNCF Contributor Strategy Technical Advisory Group

<img src="website/static/images/tag-contributor-strategy-icon-color.png" alt="a line drawing of a busy bee" align=right width="150px">

Contributor Strategy is a [Technical Advisory Group](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) (TAG) responsible for contributor experience,
sustainability, governance, and openness guidance to help CNCF community groups
and projects with their own contributor strategies for a healthy project.
Technical Advisory Groups report to the CNCF Technical Oversight Committee (TOC) providing high-quality technical expertise, unbiased information and proactive leadership in their respective area.
```

Reply:
Yes, that did not look right.
Updated. Thanks